delivery/chart/test.sh

```diff
@@ -24,12 +24,14 @@ if [[ -n "${github_token}" && -n "${github_user}" ]]; then
 fi
 
 if [[ -z "${RELEASE_BUILD}" ]]; then
-    # replace ghcr.io/podtato-head/body with ghcr.io/podtato-head/<github_user>/body for tests
+    # replace ghcr.io/podtato-head/body with ghcr.io/podtato-head/<github_user>/body for tests and test changing hat part number
     helm upgrade --install podtato-head ${this_dir} \
         --set "images.repositoryDirname=ghcr.io/${github_user:+${github_user}/}podtato-head" \
+        --set "hat.env[0].name=PODTATO_PART_NUMBER" --set "hat.env[0].value=02" \
```

Comment:
We could expand the `scripts/test_services.sh` script to look in the returned file and see that it's the expected one, but that would require some coordination between the two scripts; this is great for now. Thanks!